en/faq.md

```diff
@@ -0,0 +1,77 @@
+## FAQ
+
```

Comment:
Any other things that might be worth "asking" here - i.e any other features/expectations to set?

Might not be the right place, but I'm thinking "can I do mavlink passthrough"?

Reply:
Good idea.

Reply:
Done.